css/styles.css

```diff
@@ -0,0 +1,144 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 17px;
+}
+
+body {
+    color: #2c3340;    
+    line-height: 1.41em;
+    background-color: #6f23ff;
+}
+
+input, button, select, textarea {
+    margin: 0;
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+}
+
+.header {
+    padding: 0 0.625rem;
+    margin-bottom: 1.875rem;
+    color: #fff;
+}
+
+main {
+    width: 100%;
+    margin: 3.125rem auto 3.125rem auto;
+    max-width: 720px;
+    
+}
+
+h1 {
+    margin-bottom: 0.5rem;
+    font-weight: 400;
+    line-height: 1.2;
+    font-weight: bold;
+}
+
+p {
+    margin-bottom: 0.5rem;
+    margin-top: 0;
+    font-size: 1.125rem;
+}
+
+.text-center {
+    text-align: center;
+}
+
+form {
+    padding: 2.5rem;
+    background: #fff;
+    border-radius: 0.25rem;
+    box-shadow: 2px 2px 4px 0 rgba(0, 0, 0, 0.1);
+}
+
+.form-group {
+    margin: 0 auto 1.25rem auto;
+    padding: 0.25rem;
+}
+
+label {
+    display: flex;
+    align-items: center;
+    font-size: 1.125rem;
+    margin-bottom: 0.5rem;
+}
+
+.form-control {
+    display: block;
+    width: 100%;
+    height: 2.375rem;
+    padding: 0.375rem 0.75rem;
+    color: #495057;
+    background-color: #fff;
+    background-clip: padding-box;
+    border: 1px solid #ced4da;
+    border-radius: 0.25rem;
+}
+
+.clue {
+    margin-left: 0.25rem;
+    font-size: 0.9rem;
+}
+
+.input-radio, .input-checkbox {
+    display: inline-block;
+    margin-right: 0.625rem;
+    min-height: 1.25rem;
+    min-width: 1.25rem;
+}
+
+.input-text {
+    margin-bottom: 0.5rem;
+    margin-top: 0.5rem;
+    font-size: 0.9rem;
+}
+
+.input-textarea {
+    min-height: 120px;
+    width: 100%;
+    padding: 0.625rem;
+}
+
+button {
+    border: none;
+}
+
+.button-container {
+    display: flex;
+    justify-content: center;
+}
+
+.submit-button {
+    background-color: #41d3bd;
+    box-shadow: 1px 1px 4px 0 rgb(0 0 0 / 10%);
+    min-width: 250px;
+    padding: 1rem 0 1rem 0;
+    margin: 0.5rem auto;
+    border-radius: 4px;
+    color: #fff;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
- [ ] Plenty of empty lines at the end of this .css file, only one is necessary.

Reply:
- [ ] I think there's plenty of empty lines at the end of this .css file, only one is necessary. This helps to keep code organised and consistent.
 
You can also avoid this type of mistake by adding [Linters](https://github.com/microverseinc/linters-config) which will flag programming errors, bugs, stylistic errors, and suspicious constructs